features/step_definitions/admin_console.rb

```diff
@@ -27,6 +27,7 @@
                                password: user.password,
                                idp: env.idp)
   raise "cannot login to cluster console" unless @result[:success]
+  step "I run the :navigate_to_admin_console web action"
```

Comment:
Added step to switch to admin console for every user @yanpzhan can you see any potential issue for this update?

Reply:
No harm from my view for now.